backend/src/getUsers.js

```diff
@@ -83,7 +83,11 @@ router.post('/isAdmin', async (req, res) => {
     }
   }).catch((err) => {
     console.log(err);
-    res.status(404).send(err);
+    try {
```

Comment:
In the isAdmin endpoint can't take in a token in the beginning if the user isn't logged in since there is no valid token to verify.

Reply:
<img width="137" alt="Screen Shot 2022-05-26 at 9 22 39 AM" src="https://user-images.githubusercontent.com/67285293/170530889-3c49b100-5dd0-4887-ad10-9f4ef5aab7f7.png">
<img width="131" alt="Screen Shot 2022-05-26 at 9 23 05 AM" src="https://user-images.githubusercontent.com/67285293/170530920-30396ebf-b6ac-4d59-9772-cf8f96762fa7.png">
Can we change the underline to be across the whole text and not cut off? I think we should also change the color to that dark blue that the Woods Humane Society logo has

Reply:
For the isAdmin endpoint since there isn't a valid token to check when the user is logging in or signing up, we should maybe check localStorage for a valid token before we send it or just delete it entirely

Reply:
See latest commit!